src/vm/excep.cpp

```diff
@@ -12327,20 +12327,25 @@ VOID ThrowBadFormatWorker(UINT resID, LPCWSTR imageName DEBUGARG(__in_z const ch
 #ifndef DACCESS_COMPILE
     SString msgStr;
 
-    if ((imageName != NULL) && (imageName[0] != 0))
```

Comment:
Could you please also try some of the cases that went through this path before your change - to make sure that they look as good or better?

Reply:
I did some experimentation in the debugger. It just moves the path from the start into the message. I have fixed a mistake in the fallback path, though.